help.html

```diff
@@ -405,7 +405,6 @@ <h3>What next?</h3>
     <!-- Bootstrap core JavaScript
     ================================================== -->
     <!-- Placed at the end of the document so the pages load faster -->
-    <script src="static/js/jquery-2.1.4.min.js"></script>
```

Comment:
Removed from here since it is also included in line 23:
https://github.com/bbcmicrobit/PythonEditor/blob/e34ed3ac43aba8ea6ef15f4e57e117d1581caff4/help.html#L23